home/import_assessments.py

```diff
@@ -398,15 +406,19 @@ def from_flat(cls, row: dict[str, str], row_num: int) -> "AssessmentRow":
             )
 
 
-def get_content_page_id_from_slug(slug: str) -> int:
+def get_content_page_id_from_slug(slug: str, locale: str) -> int:
```

Comment:
Instead of sending the language code here, and then doing a join in the db, why not just send the actual locale object to this function? i.e. `locale: Locale`

Then calling the function becomes `get_content_page_id_from_slug(self.low_result_page, self.locale)`, and the filter becomes `ContentPage.objects.get(slug=slug, locale=locale)`, simplifying it.